selenium-pages/src/test/java/com/epam/engx/selenium/pages/google/GoogleCloudTest.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.engx.selenium.pages.google;
+
+import com.epam.engx.selenium.pages.browser.Browser;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+class GoogleCloudTest {
+
+    private static Browser browser;
+
+    @BeforeAll
+    static void setUp() {
+        browser = Browser.chrome();
+    }
+
+    @AfterAll
+    static void tearDown() {
+        browser.quit();
+    }
+
+    @Test
+    void goTo() {
```

Comment:
Please specify where do you go in the method's name

Reply:
done